examples/README.md

```diff
@@ -20,19 +20,18 @@ In a new terminal, run:
 $ npx hardhat node
 ```
 
-3. Run the web server
+3. Run the example in a browser
+(take note that you should hit "npm run clean" after hitting "npm run browser")
```

Comment:
Is this note still relevant? If so, I'm thinking we can rephrase it to 'Try `npm run clean` if you've made changes to the code but the web page is not updated'